gdpr/anonymizers/base.py

```diff
@@ -89,6 +89,20 @@ def get_value_from_version(self, obj, version, name: str, encryption_key: str, a
                 obj, get_reversion_local_field_dict(version)[name], encryption_key
             )
 
+    def get_value_from_entry(self, obj, entry, name: str, encryption_key: str, anonymization: bool = True):
+        def process_value(value, is_anonymized: bool):
+            if value == "None":
+                return "None"
```

Comment:
Hmm, does `auditlog` really store `None` quoted?

Reply:
yes, it does, covered in tests